src/streamlink/plugins/ine.py

```diff
@@ -23,7 +23,7 @@ class INE(Plugin):
             validate.all(
                 validate.get(1),
                 validate.transform(json.loads),
-                {"playlist": str},
+                {"playlist": unicode},
```

Comment:
```suggestion
                {"playlist": validate.text},
```

This will work on all supported Python versions, not only 2.7.